src/EnergyPlus/api/datatransfer.h

```diff
@@ -81,6 +81,15 @@ ENERGYPLUSLIB_API const char * listAllAPIDataCSV();
 ///          Do not call for variable, meter, actuator, or any other internal exchange data prior to this returning true.
 /// \return Returns 0 (success) once the data is ready, otherwise returns 1.
 ENERGYPLUSLIB_API int apiDataFullyReady();
+/// \brief Provides a user-facing check on the API error flag
+/// \details Some API functions return a value of 0, which could potentially indicate an error, or an actual 0 value.
+///          This function provides a way to disambiguate the response value.
+/// \return Returns 0 (success) if the error flag is not activated, or 1 if the error flag is true.
+ENERGYPLUSLIB_API int apiErrorFlag();
```

Comment:
Functions do not attempt to throw anymore.  For the few that did attempt a throw, they return zero, and the client is then directed to check the error flag status to disambiguate between real zero outputs and error values.  This is a very common approach.